res/css/views/right_panel/_ThreadPanel.pcss

```diff
@@ -118,6 +118,27 @@ limitations under the License.
         &.mx_EventTile_clamp:hover {
             cursor: pointer;
         }
+
+        .mx_ReactionsRow {
+            margin-bottom: 0;
+        }
+
+        .mx_EventTile_msgOption {
+            float: none;
+            width: auto;
+            height: auto;
+            line-height: 0;
+
+            .mx_ReadReceiptGroup {
+                inset-block-start: 0;
+            }
+        }
+    }
+
+    .mx_EventTile:not(:last-child) {
+        .mx_ReadReceiptGroup_noReceipts {
+            display: none;
+        }
     }
```

Comment:
could you give this a comment? Evidently it's trying to hide the read receipt group under certain conditions, but I'm failing to grok what those conditions are